.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -18,8 +20,8 @@ A clear and concise description of what you expected to happen.
 If applicable, add screenshots to help explain your problem.
 
 **Logs**
-If applicable, copy logs from your console here. Please *do not*
-use screenshots of logs, copy them as text.
+If applicable, copy **full** logs from your console here. Please *do not*
+use screenshots of logs, copy them as text or attach an *uncompressed* file.
```

Comment:
I would say using github [gist] is also acceptable too.

[gist]: https://gist.github.com/